README.md

```diff
@@ -8,6 +8,9 @@
 [![OpenSSF Scorecard](https://api.scorecard.dev/projects/github.com/percona/percona_pg_telemetry/badge)](https://scorecard.dev/viewer/?uri=github.com/percona/percona_pg_telemetry)
 [![Forum](https://img.shields.io/badge/Forum-join-brightgreen)](https://forums.percona.com/)
 
+> [!CAUTION]
+> This extension has been deprecated and replaced with a backwards compatibility stub.
```

Comment:
This extension has been deprecated and replaced with a backwards compatibility stub.
No telemetry data will be gathered moving forward.
The percona_pg_telemetry extension has been deprecated as its functionality no longer provides meaningful benefits and is no longer required for current use cases. 

It will not be maintained going forward and should not be used in new deployments.

Reply:
Hm, maybe I misunderstand your comment but I felt that was a bit clunky so only incoprated parts of it which I have pused now. What do you think? Do you want me to incorporate more of what you suggested?

Reply:
Reads good, thanks